Teacher/Teacher/Submissions/SubmissionsList/View/SubmissionListScreen.swift

```diff
@@ -0,0 +1,244 @@
+                        .foregroundStyle(Color.textDark)
+                        .accessibilityHidden(true)
+                    TextField(
+                        String(localized: "Search Submissions", bundle: .teacher),
+                        text: $viewModel.searchText, prompt: Text("Search", bundle: .teacher))
+                        .autocorrectionDisabled()
+                        .textInputAutocapitalization(.never)
+                        .font(.regular14)
+                        .foregroundStyle(Color.textDarkest)
+                    Spacer()
+                }
+                .padding(8)
+                .background(Color.backgroundLight, in: RoundedRectangle(cornerRadius: 10))
+                .padding(16)
+                .background(Color.backgroundLightest)
+            }
+        }
+    }
+
+    struct HeaderView: View {
+
+        @ScaledMetric private var uiScale: CGFloat = 1
+        let courseName: String
+
+        var body: some View {
+            SeparatedRow {
+                HStack {
+                    Text(courseName)
+                        .foregroundStyle(Color.textDarkest)
+                        .font(.semibold16)
+
+                    Spacer()
+                }
+                .padding(16)
+                .background(
+                    Color.backgroundLightest,
+                    in: RoundedRectangle(cornerRadius: 6)
+                )
+                .shadow(color: Color.black.opacity(0.08), radius: 2, x: 0, y: 2)
+                .padding(16)
+                .background(Color.backgroundLight)
+            }
+        }
+    }
+
+    struct SectionHeaderView: View {
+        let title: String
+        @Binding var isCollapsed: Bool
+
+        @ScaledMetric private var uiScale: CGFloat = 1
+
+        var body: some View {
+            SeparatedRow {
+                Button(action: {
+                    withAnimation {
+                        isCollapsed.toggle()
+                    }
+                }) {
+                    HStack {
+                        Text(title)
+                            .font(.semibold14)
+                            .multilineTextAlignment(.leading)
+                            .foregroundStyle(Color.textDark)
+                        Spacer()
+                        Image
+                            .arrowOpenDownLine
+                            .size(uiScale.iconScale * 16)
+                            .foregroundColor(.textDarkest)
+                            .rotationEffect(isCollapsed ? .degrees(0) : .degrees(-180))
+                    }
+                    .padding(.vertical, 8)
+                    .padding(.horizontal, 15)
+                    .background(Color.backgroundLightest)
+                }
+                .buttonStyle(.plain)
+            }
+        }
+    }
+}
+
+private extension SubmissionListSection {
+    var accessibilityLabel: Text {
+        Text(kind.title + ", " + String.localizedAccessibilityListCount(items.count))
```

Comment:
```suggestion
        Text(kind.title + ", " + String.localizedNumberOfItems(items.count))
```
Please use this one here